benchmarks/dynamo/check_hf_bert_perf_csv.py

```diff
@@ -17,7 +17,8 @@ def check_hf_bert_perf_csv(filename):
         model_name = row["name"]
         speedup = row["speedup"]
         # Reduce from 1.165 to 1.160, see https://github.com/pytorch/pytorch/issues/96530
-        if speedup < 1.160:
+        # Reduce from 1.160 to 1.145 after a transformer version upgrade, see https://github.com/pytorch/benchmark/pull/1406
+        if speedup < 1.145:
```

Comment:
During re-land, let's change it to 1.140, e.g. this action generated 1.1405 https://github.com/pytorch/pytorch/actions/runs/4420433748/jobs/7750698385